Sources/DNA/Color/UserInterfaceStyle.swift

```diff
@@ -24,7 +24,16 @@ public class Theme {
         }
     }
 
-    public static func setCurrentUserInterfaceStyle(_ userInterfaceStyle: UserInterfaceStyle) {
+    /// Needs to be called from main thread on iOS 13
+    public static func setCurrentUserInterfaceStyle(_ userInterfaceStyle: UserInterfaceStyle, in window: UIWindow?) {
+        if #available(iOS 13.0, *) {
+            #if swift(>=5.1)
+            let uiUserInterfaceStyle: UIUserInterfaceStyle = userInterfaceStyle == .dark ? .dark : .light
+            let updatedTraits = UITraitCollection(traitsFrom: [UITraitCollection.current, UITraitCollection(userInterfaceStyle: uiUserInterfaceStyle)])
+            UITraitCollection.current = updatedTraits
+            window?.overrideUserInterfaceStyle = uiUserInterfaceStyle
```

Comment:
Nice, is it actually possible to override user interface style?

Reply:
Yes, it seems to work well, you can do it for individual views/viewcontrollers too.

Originally I thought that `UITraitCollection.current = updatedTraits` would be enough but it wasn't (so now I'm not 100% that part is actually needed).